app/gui/view/component-browser/component-list-panel/icons/src/common_part.rs

```diff
@@ -57,18 +57,24 @@ pub fn arc(outer_radius: f32, stroke_width: f32, start_angle: f32, end_angle: f3
     (circle * angle).into()
 }
 
-/// The shape of a table, given by a grid with size `columns` x `rows`. The stroke width is 1.0 and
-/// the cell size 4.0. The origin is at the lower left corner.
-pub fn table(columns: i32, rows: i32) -> AnyShape {
-    const STROKE_WIDTH: f32 = 1.0;
-    const CELL_SIZE: f32 = 4.0;
+/// An 2d-array of squares of size `cell_size`.
+pub fn table(columns: i32, rows: i32, cell_size: f32) -> AnyShape {
+    const GAP: f32 = 1.0;
```

Comment:
This function was changed entirely because the design doesn't use the old `table`'s look.